### PR TITLE
a couple localization updates

### PR DESCRIPTION
**Pull request by Adaptivity**
- Replaced cutText with call dayz_rollingMessages
- Changed a couple bulk names to more better looking

### CODE REVIEW

**Comment by Adaptivity** [comment]
There are merge conflict but I dunno where. Please point me to it.

**Comment by ebaydayz** [comment]
The conflicts are in z_at_allowBuying.sqf and maintain_areaSC.sqf from my commit here:
https://github.com/EpochModTeam/DayZ-Epoch/commit/2f98fc04514cf174c25929884c49d6ea7d2c4c16

**Comment by Adaptivity** [comment]
Strange I didn't even changed lines which you have changed. Will revert them now.

**Comment by Adaptivity** [comment]
Well, it's not. Conflict somewhere else or Github is derped.

**Comment by Adaptivity** [comment]
Fixed.

File: SQF/dayz_code/stringtable.xml [hunk 1]
```diff
@@ -15755,44 +15756,81 @@
...
+			<English>Price to maintain: %1 %2</English>
+		</Key>
+		<Key ID="STR_EPOCH_PLOTMANAGEMENT_NO_MONEY_NEEDED">
+			<English>No money needed.</English>
+			<Russian>Деньги не требуются.</Russian>
+		</Key>
+		<Key ID="STR_EPOCH_PLOTMANAGEMENT_OBJECTS_MAINTAINED_SUCCESS">
+			<English>SUCCESS : Objects maintained: %1</English>
+		</Key>
+		<Key ID="STR_EPOCH_PLOTMANAGEMENT_PRICE_MAINTAINED_SUCCESS">
```
Comment (ebaydayz) [inline SQF/dayz_code/stringtable.xml]: Old string doesn't match. Should be:

SUCCESS : Price maintained: %1 %2 !

File: SQF/dayz_code/actions/plotManagement/maintain_area.sqf [hunk 2]
```diff
@@ -98,28 +98,28 @@ switch _option do {
...
 			_ctrl = ((uiNamespace getVariable "PlotManagement") displayCtrl 7012);
-			_result =  format["FAILED : Objects maintained: 0" , _count];
+			_result =  format[localize "STR_EPOCH_PLOTMANAGEMENT_OBJECTS_MAINTAINED_FAILED", _count];
 			_ctrl ctrlSetText   _result;			
 			_ctrl = ((uiNamespace getVariable "PlotManagement") displayCtrl 7013);
-			_result =  format["FAILED : Money needed: %1 %2 !" , (_requirements select 0) select 1, (_requirements select 0) select 0];
+			_result =  format[localize "STR_EPOCH_PLOTMANAGEMENT_MONEY_NEEDED_FAILED", (_requirements select 0) select 1, (_requirements select 0) select 0];
 			_ctrl ctrlSetText   _result;			
-			cutText [format[(localize "STR_EPOCH_ACTIONS_6"), _missingQty, _textMissing], "PLAIN DOWN"];
+			format[localize "STR_EPOCH_ACTIONS_6", _missingQty, _textMissing], call dayz_rollingMessages;
```
Comment (ebaydayz) [inline SQF/dayz_code/actions/plotManagement/maintain_area.sqf]: Shouldn't be a comma here:
<code>, call dayz_rollingMessages;</code>

**Pull request merged**